os-java-run/added/proxy-options

```diff
@@ -42,7 +42,7 @@ proxy_options() {
 
   local noProxy="${no_proxy:-${NO_PROXY}}"
   if [ -n "$noProxy" ] ; then
-    ret="$ret -Dhttp.nonProxyHosts=\"$(echo $noProxy | sed -e 's+,+|+g')\""
+    ret="$ret -Dhttp.nonProxyHosts=\\\"$(echo $noProxy | sed -e 's+,+|+g')\\\""
```

Comment:
I'm not sure this will produce the desired results with the sso images (which also use this functionality).  It might be better to address this directly in the amq launch.sh script, specifically by escaping all quotes in ACTIVEMQ_OPTS when writing the env file.